packages/vitest/src/node/config/resolveConfig.ts

```diff
@@ -677,7 +677,7 @@ export function resolveConfig(
   if (resolved.cache !== false) {
     let cacheDir = VitestCache.resolveCacheDir(
       '',
-      resolve(viteConfig.cacheDir, 'vitest'),
+      viteConfig.cacheDir,
```

Comment:
`vitest` suffix is to avoid collision of `node_modules/.vite/deps` directory, so using it for `results.json` is not technically necessary.